Documentation/TopLevelObjects/Module.rst

```diff
@@ -26,108 +27,125 @@ In simple backend modules extension authors can decide how to use this
+..  _tlo-module-properties-partialRootPaths:
 
 view.partialRootPaths
 ----------------------
 
-.. rst-class:: dl-parameters
+..  confval:: view.partialRootPaths.[array]
+    :name: module-view-partialRootPaths
+    :type: file path with :ref:`stdWrap <stdwrap>`
+
+    Used to define several paths for partials, which will be executed in reverse
+    order. The first folder where the desired partial is found, is used. The
+    keys of the array define the order.
 
-view.partialRootPaths
-   :sep:`|` :aspect:`Data type:` array of file paths with :ref:`stdWrap <stdwrap>`
-   :sep:`|`
 
-   Used to define several paths for partials, which will be executed in reverse
-   order. The first folder where the desired partial is found, is used. The
-   keys of the array define the order.
-   **Example:**
+Example: Set the partial root paths
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   .. code-block:: typoscript
-      :caption: EXT:my_extension/ext_typoscript_setup.typoscript
+..  code-block:: typoscript
+    :caption: EXT:my_extension/ext_typoscript_setup.typoscript
 
-      module.tx_somebackend_module {
-          view {
-              partialRootPaths {
-                  100 = EXT:my_extension/Resources/Private/Partials/Backend
-              }
-          }
-      }
+    module.tx_somebackend_module {
+        view {
+            partialRootPaths {
+                100 = EXT:my_extension/Resources/Private/Partials/Backend
+            }
+        }
+    }
+
+
+..  index:: module; settings
+..  _tlo-module-properties-settings:
 
 settings
 --------
 
+..  confval:: settings
+    :name: module-settings
+    :type: array<string, mixed>
+
 Here resides all of the settings. These settings are
-available in the Controller of the backend module as the array variable
+available in the controller of the backend module as the array variable
 :php:`$this->settings`.
 
-Example:
+Example: Limit pagination in the backend
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Show 25 news records in the backend module of the news extension:
 
-.. code-block:: typoscript
+..  code-block:: typoscript
    :caption: EXT:my_extension/ext_typoscript_setup.typoscript
 
    module.tx_news {
       settings.list.paginate.itemsPerPage = 25
    }
 
+Example: Register YAML file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Register your EXT:form configuration for the backend via TypoScript.
+Register your :doc:`typo3/cms-form:Index` configuration for the backend via TypoScript.
 
-.. code-block:: typoscript
-   :caption: EXT:my_extension/ext_typoscript_setup.typoscript
+..  code-block:: typoscript
+    :caption: EXT:my_extension/ext_typoscript_setup.typoscript
 
-   module.tx_form {
-       settings {
-           yamlConfigurations {
-               100 = EXT:my_extension/Configuration/Form/CustomFormSetup.yaml
-           }
-       }
-   }
+    module.tx_form {
+        settings {
+            yamlConfigurations {
+                100 = EXT:my_extension/Configuration/Form/CustomFormSetup.yaml
```

Comment:
```suggestion
                # Use the current timestamp as key to avoid accidental overwriting
                1712163960 = EXT:my_extension/Configuration/Form/CustomFormSetup.yaml
```